### PR TITLE
Work on a proper mquery package

### PR DESCRIPTION
**Pull request by msm-code**
WIP PR, I still need to update documentation etc. But I've tested this and it roughly works.

**Your checklist for this pull request**
- [x] I've read the [contributing guideline](https://github.com/CERT-Polska/mquery/blob/master/CONTRIBUTING.md).
- [x] I've tested my changes by building and running mquery, and testing changed functionality (if applicable)
- [ ] I've added automated tests for my change (if applicable, optional)
- [ ] I've updated documentation to reflect my change (if applicable)

**What is the current behaviour?**
No python package

**What is the new behaviour?**
Package installable with `pip` is available

### CODE REVIEW

**Comment by msm-code** [comment]
Tests fixed, now just fix dev dockerfiles, and figure out how to build it

`include_front` in setup.py is a temporary hack, we should probably build this along with the rest of the package? But how? TODO check how mwdb-core does this.

Also refactor the dockerfiles, we just need to `pip install .` at this point and everything should work out.

File: deploy/docker/dev.web.Dockerfile [hunk 1]
```diff
@@ -8,5 +8,4 @@ RUN apt update; apt install -y cmake
 COPY requirements.txt src/plugins/requirements-*.txt /tmp/
 RUN ls /tmp/requirements*.txt | xargs -i,, pip --no-cache-dir install -r ,,
 
-# ./src is expected to be mounted with a docker volume
-CMD ["uvicorn", "app:app", "--host", "0.0.0.0", "--port", "5000", "--reload"]
+CMD pip install -e /usr/src/app && uvicorn mquery.app:app --host 0.0.0.0 --port 5000 --reload
```
Comment (msm-code) [inline deploy/docker/dev.web.Dockerfile:11]: This is only a dev dockerfile, we need to install the pip package in editable mode to have hot reload.

File: deploy/docker/dev.frontend.Dockerfile [hunk 2]
```diff
@@ -4,6 +4,5 @@ RUN npm install -g serve
 COPY src/mqueryfront /app
 WORKDIR /app
 RUN npm install --legacy-peer-deps && \
-    mkdir ./public/monaco-vs && \
-    cp -r ./node_modules/monaco-editor/min/vs/. ./public/monaco-vs
+    cp -r ./node_modules/monaco-editor/min/vs ./public/monaco-vs
```
Comment (msm-code) [inline deploy/docker/dev.frontend.Dockerfile:7]: this cp step will be fixed in a follow-up PR anyway.

File: deploy/docker/web.Dockerfile [hunk 3]
```diff
@@ -4,20 +4,18 @@ RUN npm install -g serve
 COPY src/mqueryfront /app
 WORKDIR /app
 RUN npm install --legacy-peer-deps && \
-    mkdir ./public/monaco-vs && \
-    cp -r ./node_modules/monaco-editor/min/vs/. ./public/monaco-vs && \
+    cp -r ./node_modules/monaco-editor/min/vs ./public/monaco-vs && \
```
Comment (msm-code) [inline deploy/docker/web.Dockerfile:7]: this cp step will be fixed in a follow-up PR anyway.

**Review by nazywam — APPROVED** [review]

File: deploy/docker/dev.daemon.Dockerfile [hunk 4]
```diff
@@ -8,5 +8,4 @@ RUN apt update; apt install -y cmake
 COPY requirements.txt src/plugins/requirements-*.txt /tmp/
 RUN ls /tmp/requirements*.txt | xargs -i,, pip --no-cache-dir install -r ,,
 
-# ./src is expected to be mounted with a docker volume
-CMD ["./autoreload", "python3", "daemon.py"]
+CMD pip install -e /usr/src/app && mquery-daemon
```
Comment (msm-cert) [inline deploy/docker/dev.daemon.Dockerfile:11]: (this broke autoreload)

**Pull request merged**